### PR TITLE
Fix tctl Database CA compatibility with v9 cluster

### PR DESCRIPTION
**Pull request by jakule**
`tctl` currently fails (with the below error) when talking to v9 cluster as it tries to get Database CA which is not available in v9.
```
ERROR: 'db' authority type is not supported
```

This PR changes the error to warning log for:
```
tctl status
tctl get cert_authority
```
and excludes Database CA from being fetched by default for:
```
tctl auth export
```
Database CA was added here as a part of the main Database CA PR https://github.com/gravitational/teleport/pull/9593. `tctl auth export --type=db` can still print the Database CA.

To reviewers: there is a few ways of fixing that problem. I'm open to suggestions if you think that this solution can cause some problems in the future.

### CODE REVIEW

File: tool/tctl/common/auth_command.go [hunk 1]
```diff
@@ -176,9 +176,9 @@ func (a *AuthCommand) ExportAuthorities(ctx context.Context, client auth.ClientI
 		return a.exportTLSAuthority(ctx, client, types.UserCA, true)
 	}
 
-	// if no --type flag is given, export all types
+	// if no --type flag is given, export HostCA and UserCA.
 	if a.authType == "" {
-		typesToExport = []types.CertAuthType{types.HostCA, types.UserCA, types.DatabaseCA}
```
Comment (smallinsky) [inline tool/tctl/common/auth_command.go:181]: I think that `types.DatabaseCA` should be still present in `typesToExport` list. If we remove it there won't be ability to export it.

**Comment by Tener** [comment]
Can we simply take the server version into account? This should be reasonably easy to do given `tctl status` already queries the server version. We can make `tctl auth export` do the same.

Alternatively, we should be able to detect "not supported" error?

Ideally, we shouldn't display warnings about stuff that is known not to work.

**Comment by jakule** [comment]
@Tener Yes, we could check the server version and even skip the error only if `caType == DatabaseCA`, but I think in general for example `status` command shouldn't print nothing only because one of few gets failed. Being more strict in this case means that every single time when we add another CA we need to go through the same process. 
If we want to make more clear that the server doesn't support database CA we could also print something like `databaseCA: not available` instead of a log that is not visible by default.

WDYGT? CC @smallinsky

**Comment by Tener** [comment]
> If we want to make more clear that the server doesn't support database CA we could also print something like `databaseCA: not available` instead of a log that is not visible by default.

I think this is a great idea, I really like it.

File: tool/tctl/common/auth_command.go (hunk 1 above)
Comment (jakule) [reply tool/tctl/common/auth_command.go:181]: Here is the handler for `--type=db` https://github.com/gravitational/teleport/pull/13177/files#diff-1da7bc0f8a6f715d9554a2bf8b7a97551d546cb187ab2135b78df3cf7636f21dL173

**Comment by jakule** [comment]
> > If we want to make more clear that the server doesn't support database CA we could also print something like `databaseCA: not available` instead of a log that is not visible by default.
> 
> I think this is a great idea, I really like it.

@Tener Added https://github.com/gravitational/teleport/pull/13177/commits/db403f06752a74dc48dcaa26bbd7186867da53de:
```
% tctl status
Cluster  localhost.me
Version  9.2.4
host CA  never updated
user CA  never updated
jwt CA   never updated
db CA    'db' authority type is not supported
CA pin   sha256:xxxxxxxxxxxxxx
```

**Review by Tener — APPROVED** [review]
LGTM.

**Review by smallinsky — APPROVED** [review]

**Pull request merged**